enhancements/kube-apiserver/unsupported-file-sync.md

```diff
@@ -0,0 +1,153 @@
+---
+title: unsupported-file-sync
+authors:
+  - stlaz
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - tkashem
+  - deads2k
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - tkashem
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - None
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: 
+  - AUTH-387
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Unsupported file synchronization into kube-apiserver static pods
+
+## Summary
+
+Since kube-apiserver pods are running as static, it's hard to test upcoming features
+that might require configuration via files. This enhancement describes how to synchronize
+files to kube-apiserver pods in an unsupported mode that matches the testing use-case.
+
+## Motivation
+
+Simplify testing features that might require additional files to be installed into
+kube-apiserver pods.
+
+### User Stories
+
+* As a developer, I want to be able to create configuration/trust files in kube-apiserver
```

Comment:
```suggestion
* As a developer, I want to be able to create additional configuration/trust files in kube-apiserver pod configuration
```

Reply:
I might go with `As a developer, I want to be able to create additional configuration/trust files inside of the kube-apiserver pod` if that's ok

Reply:
yeah, that sounds good to me